src/components/benefit-cta-link.vue

```diff
@@ -63,17 +63,23 @@ export default {
     },
   },
   methods: {
+    hasFinishedSimulation() {
+      return this.store.calculs.resultats._id
+    },
+
     getURL(link) {
+      if (!this.hasFinishedSimulation()) {
+        return this.benefit.link
```

Comment:
Plusieurs remarques :
- `this.benefit.link` est différent de `link`.
- En faisant cela, on prive les usagers de la diversité des liens `link`/`teleservice`/`instructions`/`form`.